cookbooks/ey-base/recipes/custom.rb

```diff
@@ -0,0 +1,3 @@
+if fetch_env_var(node, "EY_REDIS_ENABLED")
```

Comment:
Let's check if it's actually set to `/^TRUE$/i`.

Reply:
Also, see https://github.com/engineyard/ey-cookbooks-stable-v6/pull/113

Reply:
I added a commit for true. Still working on `include_recipe_if_enabled('component-recipe', 'COMPONENT')` if we can add that.